README.md

```
@@ -480,22 +482,26 @@ Since the keys should be stored within the DefraDB data and configuration direct
 If not saved under `~/.defradb/certs` then the public (`pubkeypath`) and private (`privkeypaths`) key paths need to be explicitly defined in addition to the `--tls` flag or `tls` set to `true` in the config.
 
 Then to start the server with TLS, using your generated keys in custom path:
+
 ```shell
 defradb start --tls --pubkeypath ~/path-to-pubkey.key --privkeypath ~/path-to-privkey.crt
 
 ```
 
 ## Access Control System
+
 Read more about the DefraDB ACP System [here](/acp/README.md)
```

markdownlint-cli2 (0.20.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)